src/host/load_monitor.h

```diff
@@ -65,33 +65,36 @@ namespace asynchost
 
     void on_timer()
     {
-      const auto message_counts = dispatcher.retrieve_message_counts();
-      const auto time_now =
-        std::chrono::duration_cast<std::chrono::milliseconds>(
-          TClock::now().time_since_epoch());
-
-      if (!message_counts.empty())
+      if (ccf::logger::config().level() <= ccf::LoggerLevel::DEBUG)
       {
-        auto j = nlohmann::json::object();
-
-        j["start_time_ms"] = last_update.count();
-        j["end_time_ms"] = time_now.count();
+        const auto message_counts = dispatcher.retrieve_message_counts();
+        const auto time_now =
+          std::chrono::duration_cast<std::chrono::milliseconds>(
+            TClock::now().time_since_epoch());
 
+        if (!message_counts.empty())
         {
-          j["ringbuffer_messages"] =
-            dispatcher.convert_message_counts(message_counts);
+          auto j = nlohmann::json::object();
 
-          LOG_DEBUG_FMT("Host load: {}", j.dump());
-        }
+          j["start_time_ms"] = last_update.count();
+          j["end_time_ms"] = time_now.count();
 
-        {
-          j["ringbuffer_messages"] = enclave_counts;
-          enclave_counts = nlohmann::json::object();
+          {
+            j["ringbuffer_messages"] =
+              dispatcher.convert_message_counts(message_counts);
 
-          LOG_DEBUG_FMT("Enclave load: {}", j.dump());
-        }
+            LOG_DEBUG_FMT("Host load: {}", j.dump());
```

Comment:
`LOG_DEBUG_FMT` contains an early-out, and will avoid dumping this object if the configured level is `INFO` or higher. But we've already:

- Looked up and incremented a count for every message sent
- Serialised the current counts map to JSON on every enclave tick
- Dumped that JSON to enclave on the ringbuffer
- Deserialised that JSON on the host, from the ringbuffer
- Manually merged that JSON with an existing JSON object, tracking the enclave work since the last `LoadMonitor` tick
- Then every 500ms, we potentially print these

So the short-circuit wasn't saving much!

We should of course stop doing _all_ of this, as we look at stripping the ringbuffer out entirely.